server/service/integration_mdm_dep_test.go

```diff
@@ -122,11 +122,11 @@ func (s *integrationMDMTestSuite) TestDEPEnrollReleaseDeviceGlobal() {
 	s.enableABM("fleet_ade_test")
 
 	// test manual and automatic release with the new setup experience flow
-	for _, enableReleaseManually := range []bool{false, true} {
-		t.Run(fmt.Sprintf("enableReleaseManually=%t", enableReleaseManually), func(t *testing.T) {
-			s.runDEPEnrollReleaseDeviceTest(t, globalDevice, enableReleaseManually, nil, "I1", false)
-		})
-	}
+	// for _, enableReleaseManually := range []bool{false, true} {
+	// 	t.Run(fmt.Sprintf("enableReleaseManually=%t", enableReleaseManually), func(t *testing.T) {
+	// 		s.runDEPEnrollReleaseDeviceTest(t, globalDevice, enableReleaseManually, nil, "I1", false)
+	// 	})
+	// }
```

Comment:
Is there a reason to comment out these tests (and the ones beginning at line 221 below)?

Reply:
derp! I forgot to remove them (we don't need them anymore)